srcpkgs/emptty/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'emptty'
+pkgname=emptty
+version=0.1.0
+revision=1
+archs="x86_64"
+build_style=gnu-makefile
+make_build_target=build
+make_install_target="install install-config install-runit"
+makedepends="go gcc pam-devel"
```

Comment:
`go` and `gcc` are not required with `build_style=go` (and `gcc` is not required even in the current template because it should already be installed in the `masterdir`).